.storybook/components/DesignTokens/Legacy/LegacyColor.stories.mdx

```diff
@@ -3,54 +3,31 @@ import { Meta, Preview } from '@storybook/addon-docs';
 import TokenTable from './TokenTable.tsx';
 import tokens from '../../../../src/tokens-dist/json/variables-nested.json';
 
-<Meta title="Color Tokens" />
+<Meta title="Legacy Color Tokens" />
```

Comment:
Now this story only shows tokens that are no longer supported in the Figma kit:
<img width="995" alt="Screen Shot 2022-05-18 at 12 48 33 PM" src="https://user-images.githubusercontent.com/15840841/169101279-03b1d64e-5920-4194-a9f2-b167be3f800c.png">

To minimize change I'm deciding to keep these around until we figure out a deprecation plan